DEVELOPMENT.md

```diff
@@ -99,6 +99,8 @@ helmfile -e <service|workload>_cluster -lapp=ingress-nginx -lapp=node-local-dns
 
 > [!important]
 > To use certificates from Let's Encrypt you must enable [DNS-01 challenges in `cert-manager`](https://cert-manager.io/docs/configuration/acme/dns01/).
+> Remember to add the correct network policies.
+> For Amazon route53 the ip range is: `205.251.192.0/18`
```

Comment:
Style
```suggestion
> Remember to add the correct network policies, for Amazon Route 53 the IP range is: `205.251.192.0/18`.
```

Also is this range region specific, and what does it point to API or DNS resolvers?

Reply:
Unsure, it was looking up several IP addresses. One came back as `AMAZON-05`, with no region name.